.travis.yml

```diff
@@ -3,7 +3,7 @@ node_js: 10
 services:
   - xvfb
 addons:
-  firefox: '51.0'
+  firefox: latest
```

Comment:
I’m not sure what Firefox 51 is missing that we need, but I had to bump the version to get the build working.

Reply:
It appears to be throwing on this check for `async function`: https://github.com/canjs/can-observable-mixin/blob/b679ca0586f5e8595197c82db24e4c78773062cf/src/define.js#L44

Looks like async function support was added in Firefox 52.

Reply:
Firefox 52 was released like 2.5 years ago. There shouldn't be much risk of people running into this problem.